### PR TITLE
ruby sdk

### PR DESCRIPTION
**Pull request by zaza81**

### CODE REVIEW

File: src/master/jumpstart.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ SDKs make it comfortable to talk to the server from your favourite language, wit
 The most well-known immudb SDK is written in [Golang](https://golang.org/), but there are SDKs available for Python, NodeJS, Java and others.
 
 For other unsupported programming languages, [immugw](/master/immugw/) provides a REST gateway that can be used to talk to the server via generic HTTP.
+ 
+The [Ruby](https://github.com/ankane/immudb-ruby) SDK is maintained by the community, credits to [Ankane](https://github.com/ankane).
```
Comment (byo) [inline src/master/jumpstart.md:39]: We should have some table with SDKs for programming languages with links to those SDK docs (with exception of golang SDK where it is documented on docs.immudb.io), most likely we would split this into two parts where officially supported SDKs are listed first followed by community maintained ones.

**Review by byo — CHANGES_REQUESTED** [review]

File: src/master/jumpstart.md (hunk 1 above)
Comment (zaza81) [reply src/master/jumpstart.md]: To me it does not make senso to split right now as we just have one SDK maintainer by the community. I added a column make it clear

File: src/master/jumpstart.md [hunk 2]
```diff
@@ -22,19 +22,39 @@ In the most common scenario, you would perform write and read operations on the
...
 ## SDKs
 
+SDKs make it comfortable to talk to the server from your favourite language, without having to deal with details about how to talk to it.
+
+The most well-known and recommended immudb SDK is written in [Golang](https://golang.org/), but there are other SDKs available, both maintainer by the internal team and by the community.
+
+
+| Language         | Maintainer | Immdb version | link | Notes                                                                              |
+|-------------------|---------|------------------|-------------|-----------------------------------------------------------|
+| `go`               | immudb team  | 1.2.1       |     [link](https://github.com/codenotary/immudb)  |                                   |
```
Comment (byo) [inline src/master/jumpstart.md:32]: I'm wondering if it wouldn't be better to link to https://pkg.go.dev/github.com/codenotary/immudb/pkg/client - the whole immudb repository maybe too large and people may get confused.

**Review by byo — APPROVED** [review]
Just one comment about the go sdk link, other than that LGTM

**Pull request merged**